test/IntegrationTests/NServiceBusTests.cs

```diff
@@ -0,0 +1,82 @@
+// <copyright file="NServiceBusTests.cs" company="OpenTelemetry Authors">
+// Copyright The OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// </copyright>
+
+using IntegrationTests.Helpers;
+using OpenTelemetry.AutoInstrumentation.Configuration;
+using Xunit;
+using Xunit.Abstractions;
+
+namespace IntegrationTests;
+
+public class NServiceBusTests : TestHelper
+{
+    public NServiceBusTests(ITestOutputHelper output)
+        : base("NServiceBus", output)
+    {
+        EnableBytecodeInstrumentation();
+    }
+
+    [Fact]
+    [Trait("Category", "EndToEnd")]
+    public void SubmitsTraces()
+    {
+        using var collector = new MockSpansCollector(Output);
+        SetExporter(collector);
+        collector.Expect("NServiceBus.Core");
+        SetEnvironmentVariable(ConfigurationKeys.Metrics.MetricsEnabled, bool.FalseString);
```

Comment:
Why?

Reply:
To have a proof that Traces can work without Metrics.

Reply:
How about adding a comment?

```suggestion
         SetEnvironmentVariable(ConfigurationKeys.Metrics.MetricsEnabled, bool.FalseString); // make sure that metrics instrumentation is not needed
```